apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/DocumentParams/HistoryLogParams/index.tsx

```diff
@@ -0,0 +1,14 @@
+import {
+  PlaygroundInput,
+  PlaygroundInputs,
+} from '$/hooks/useDocumentParameters'
+
+export function HistoryLogParams({
+  inputs: _inputs,
+  setInput: _setInput,
+}: {
+  inputs: PlaygroundInputs
+  setInput: (param: string, value: PlaygroundInput) => void
+}) {
+  return <div>TODO: DatasetParams</div>
```

Comment:
this ain't gonna be visible to users no?

Reply:
No. Check the entry point